src/pages/iou/IOUCurrencySelection.js

```diff
@@ -41,11 +41,9 @@ class IOUCurrencySelection extends Component {
     constructor(props) {
         super(props);
 
-        const {currencyOptions} = OptionsListUtils.getCurrencyListForSections(this.getCurrencyOptions(this.props.currencyList), '');
-
         this.state = {
             searchValue: '',
-            currencyData: currencyOptions,
+            currencyData: this.getFilteredCurrencyList(),
```

Comment:
```suggestion
            currencyData: this.getCurrencyOptions(this.props.currencyList),
```
This is similar to was I was fighting the contributor on: let's not call a function with a parameter with a specific value such that the function does nothing

Reply:
Oh yes, got it. There is then no need for a function for the filtering, since it is only called once.